editor/assets/stylesheets/_variables.scss

```diff
@@ -19,6 +19,7 @@ $light-gray-300: #edeff0;
 $light-gray-200: #f3f4f5;
 $light-gray-100: #f8f9f9;
 $white: #fff;
+$red: #dc3232;
```

Comment:
There's already an $alert-red color. I wonder if we should use it and update its hex value to match this.